### PR TITLE
Release 4.9.0

### PR DESCRIPTION
**Pull request by nim65s**
:warning: **Please don't merge** :warning:

ref https://github.com/humanoid-path-planner/hpp-core/pull/211

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #
-# Copyright (c) 2014 LAAS-CNRS
-# Authors: Mathieu Geisert
+# Copyright (c) 2014, 2020 LAAS-CNRS
+# Authors: Mathieu Geisert, Guilhem Saurel
```
Comment (jmirabel) [inline CMakeLists.txt:3]: Well, since here, you may also add my name :)

Comment (nim65s) [reply CMakeLists.txt]: Of course :)

**Comment by jmirabel** [comment]
Can the binary on robotpkg include pyqcustomplot and pyqgv plugins ?
They must be turned on with a CMake variable. Have a look at *plugin/CMakeLists.txt*.

**Comment by nim65s** [comment]
for pyqcustomplot & pyqgv, this require PythonQt, which is not well packaged yet.

Are you using https://github.com/gepetto/pythonqt ?

I think that if we manage to use https://github.com/MeVisLab/pythonqt instead, it would be really better. If this only requires writing and testing a `FindPythonQt.cmake`, I can handle that.

If we really need our fork for other reasons than packaging, I think we should either try to contribute our modifications to the upstream repo, or properly fork it and change its name to avoid collisions.

**Comment by jmirabel** [comment]
I used PythonQt from https://github.com/commontk/PythonQt. I tried to get my changes accepted but couldn't.

If I remember well, the differences (with https://github.com/commontk/PythonQt) are only some CMake stuff. The binaries, as of a few years ago, did not initialize everything. This, however, can easily be fixed if we know what should be initialized. By that I mean that we shouldn't rely on the PythonQt_QtAll extension and put this file https://github.com/Gepetto/PythonQt/blob/qt4/extensions/PythonQt_QtAll/PythonQt_QtAll.cpp inside gepetto-viewer instead. Then, it is only a matter of CMake.

**Comment by nim65s** [comment]
This is not the official PythonQt repository, so this PR https://github.com/commontk/PythonQt/pull/69 can't update the official pythonqt repository.

I think the official one is https://github.com/MeVisLab/pythonqt and it still has no CMake packaging (but we don't need it).

What should I try to see if anything works as you expect ? Just compile these 2 plugins ? Or have I to use them in some way ?

**Comment by jmirabel** [comment]
You are right, we don't need it to use CMake.

If you manage to compile gepetto-gui, then, in the Python console, check that `PythonQt.QtCore` and `PythonQt.QtGui` exists (they are already loaded python modules. If you have that, then, almost all the job is done.

**Comment by jmirabel** [comment]
> If you manage to compile gepetto-gui, then, in the Python console, check that `PythonQt.QtCore` and `PythonQt.QtGui` exists (they are already loaded python modules. If you have that, then, almost all the job is done.

Of course, this could be automated, but I am not sure it is worth it.

**Pull request merged**